### PR TITLE
Add ASF Privacy Policy; remove Google Analytics

### PR DESCRIPTION
**Pull request by srowen**
We do need to decide whether to replace GA with Matomo (does anyone use it?) but need to add the privacy policy link and remove GA at least.

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -117,6 +94,7 @@
           <li><a class="dropdown-item" href="{{site.baseurl}}/powered-by.html">Powered By</a></li>
           <li><a class="dropdown-item" href="{{site.baseurl}}/committers.html">Project Committers</a></li>
           <li><a class="dropdown-item" href="{{site.baseurl}}/history.html">Project History</a></li>
+          <li><a class="dropdown-item" href="https://privacy.apache.org/policies/privacy-policy-public.html">Privacy Policy</a></li>
```
Comment (srowen) [inline _layouts/home.html:97]: This is the privacy policy; everything else is removing GA

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by dongjoon-hyun** [comment]
I agree that we can add `Matomo` later. This PR itself looks enough to meet the new ASF requirements.

**Comment by srowen** [comment]
Thank you -- see also https://github.com/apache/spark/pull/36310

**Pull request merged**